### ISSUE TITLE
HTTPS/SSL support

### ISSUE THREAD

**Issue by qnorsten** [brandur/rserve#1]
Just found your project when browsing the https://github.com/ncw/rclone/wiki/Third-Party-Integrations-with-rclone page. 
I want to start by extending a great thank you for creating this application, it can be extremely useful for me.  

I just want to know if it is possible/any plans to add ssl/https support? 
I tried to look through the code, but I know close to nothing about the GO-language.

**Comment by qnorsten** [comment]
It turned out to be a really easy fix 
I hacked something together that works for me right now 
Only thing I had to do was change https://github.com/brandur/rhttpserve/blob/1453176256159c3fad886bba8aae6829603e134f/cmd/serve/serve.go#L57

into `log.Fatal(s.ListenAndServeTLS("default.crt","default.key"))`

It is an ugly hack but I guess it should be simple to add an environmental variable for that information.  In case you ever want to implement it.

**Comment by brandur** [comment]
Yep, that'll do the trick. Luckily, Go bakes TLS right into the core language, so getting support isn't too difficult.

Just a quick FYI that I stopped using Rclone for remote operations (largely after Amazon killed their unlimited cloud drive), and I don't really use this project anymore. Go's a fairly stable language so it probably still works, but please consider it largely unmaintained.

**Comment by qnorsten** [comment]
Thank you for the response and creating the project:)
It still work great, but always great to know that it is unmaintained.
I ended up adding the modification I needed into my own fork https://github.com/qnorsten/rhttpserve (SSL support + ability to customise url expiration date). 
It is probably not great (my first time coding anything in golang)
Just wanted to leave that information here in case anyone else is looking for the same functionality. 
Feel free to close this issue if you want.

**Comment by pataquets** [comment]
Hi, @brandur.
I'd like to +1 @qnorsten's thank you.
Since losing such an useful software would be sad, I'd like to suggest you posting your project on [Adoptoposs](https://adoptoposs.org/) to hopefully keep it alive. Also placing a "seeking new maintainer" message on the readme might help.

**Comment by brandur** [comment]
Thanks for the suggestion @pataquets. I'll probably wait for a little more interest before trying such though — it is probably possible to resurrect this project with some effort, but it's built with some pretty old conventions and probably needs a significant amount of work to get up to a good level of standards. Most devs would probably prefer to write a project anew rather than sink that much time into an existing one.
